apps/passport-server/test/lemonade/MockLemonadeServer.ts

```diff
@@ -0,0 +1,110 @@
+import { loadDevMessages, loadErrorMessages } from "@apollo/client/dev";
+import { stringify } from "csv-stringify/sync";
+import { MockedRequest, RequestHandler, graphql, rest } from "msw";
+import urljoin from "url-join";
+import {
+  LemonadeTicket,
+  LemonadeTicketType
+} from "../../src/apis/lemonade/lemonadeAPI";
+import { LemonadeDataMocker } from "./LemonadeDataMocker";
+
+// Apollo client doesn't load error messages by default so we have to call this
+loadDevMessages();
+loadErrorMessages();
```

Comment:
can we put these function calls into a function that is called at the beginning of the generic issuance test file? having these be in the module like this is setting off alarms for me.